shiftcommerce/reference-site
# OMS Fixes - Shift React Components updates

## Description

This PR changes the layout and styles of the order history page in the account section, to look more like the Trendy designs

## Related Issue

https://github.com/shiftcommerce/trendy-golf/issues/139

### Checklist

- [x] Appropriate commenting on functions/components (JSDoc format).
- [ ] All SHIFT library versions bumped and published (utilised in this PR).
- [x] Unit tests added/amended.
- [x] Integration tests added/amended.
- [x] Environment variables supplied (.env.example also updated)

### How has/can this been tested?

- [x] Serverside rendering
- [x] Clientside rendering

1. Login to the customer area
2. Go to Orders
3. You should see a list of orders
4. You should be able to click into each order

### Dependencies Review

N/A
